roles/monitoring_server/templates/prometheus_rules.yml.j2

```
@@ -1,4 +1,14 @@
 groups:
+  - name: tls-expiry
+    rules:
+      - alert: SSLCertExpiringSoon
+        expr: probe_ssl_earliest_cert_expiry - time() < 86400 * 14
```

```suggestion
        expr: probe_ssl_earliest_cert_expiry - time() < 60 * 60 * 24 * 14
```
just to make more explicit that this is two weeks in seconds

---

Changed as suggested.